src/sagemaker/jumpstart/factory/model.py

```diff
@@ -489,6 +549,9 @@ def _add_tags_to_kwargs(kwargs: JumpStartModelDeployKwargs) -> Dict[str, Any]:
             kwargs.tags, kwargs.model_id, full_model_version, kwargs.model_type
         )
 
+    if kwargs.hub_arn:
+        kwargs.tags = add_hub_content_arn_tags(kwargs.tags, kwargs.hub_arn)
```

Comment:
It seems it is passing the hub_arn instead of hub_content_arn to the method. 

Can be fixed after initial release, since for revenue tracking we will be needing the key part only.